js/resumeBuilder.js

```diff
@@ -24,10 +24,10 @@ var bio = {
 		$('#header').append(formattedMessage);
 		$('#header').append(HTMLskillsStart);
 
-		for(i in bio.skills) {
-			var formattedSkills = HTMLskills.replace("%data%", bio.skills[i]);
-			$('#header').append(formattedSkills);
-		}
+			for(i in bio.skills) {
```

Comment:
Creating global 'for' variable. Should be 'for (var i ...'.<br>The body of a for in should be wrapped in an if statement to filter unwanted properties from the prototype.